.github/actions/smoke-test/test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# test.sh
+#
+# Run test/test.sh in docker container using devcontainer exec.
+
+set -o errexit 
+set -o nounset 
+
+readonly TEMPLATE_ID="$1" 
+readonly SRC_DIR="/tmp/${TEMPLATE_ID}"
+
+echo "Running Smoke Test"
+
+readonly ID_LABEL="test-container=${TEMPLATE_ID}"
+devcontainer exec \
+  --workspace-folder "${SRC_DIR}" \
+  --id-label "${ID_LABEL}" \
+  /bin/sh -c '\
+    set -o errexit && \
+    if [ -f "test-project/test.sh" ]; then \
```

Comment:
i tried [[ and also == but those are bash built-in and not recognized here. @mbookman

Reply:
So in this context, is it actually running the Bourne Shell? `/bin/sh` is _typically_ aliased to `/bin/bash`, but there's obviously no requirement.

Can you instead use

```
devcontainer exec \
...
  /bin/bash -c 
...
```

or 

```
devcontainer exec \
...
  bash -c 
...
```

If the Bash path can change? I'm not that familiar with the devcontainer framework, so don't know if one can count on Bash being available in the devcontainer. A cursory glance at some Google search results makes me think you can.